### DISCUSSION TITLE
Welcome to asdf-rust Discussions!

### DISCUSSION THREAD

**Discussion by smorimoto** [asdf-community/asdf-rust#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikkosaarisalo** [comment]
From where does `cargo install` get its installation root?

I am using have asdf installed rust on MacOS.  `cargo install` does the installation into `$HOME/.asdf/installs/rust/1.68.0/bin/`.    

I am aware of the cargo installation root precedence list in https://doc.rust-lang.org/cargo/commands/cargo-install.html.  I have not set any of the higher precedence environment variables or configurations.  Therefore the installation should default to `$HOME/.cargo`, but it does not. 

Why?  How does asdf rust plugin direct the cargo installation to `$HOME/.asdf/installs/rust/1.68.0/bin/`?

**Reply by mikkosaarisalo** [reply]
After a bit of playing around I found out that `asdf reshim rust` makes the cargo installed tools visible in path - and that is cleaner solution when you want to keep things isolated between the versions.

**Reply by mikkosaarisalo** [reply]
Please advise if there is a more suitable or vibrant forum for questions like this.
